packages/components/src/templates/quote.stories.ts

```diff
@@ -17,174 +17,103 @@ export default {
+          />
           <div>
-            <p class="sd-paragraph font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm">Job Title</p>
+            <p class="sd-paragraph font-bold mb-1">Hans Joachim Reinke</p>
+            <p class="sd-paragraph sd-paragraph--size-sm">Chairman of the Board</p>
           </div>
         </div>
-        ${information()}
       </div>
-
-      <div class="text-left text-black">
+      <p class="sd-display mb-6">size 3xl</p>
+      <div class="text-left text-black mb-12">
         <q class="block mb-4 text-2xl lg:text-3xl leading-tight"
-          >Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q
+          >The environment is becoming increasingly challenging. Despite all the crises, the opportunities are greater
+          than the risks in the long term – this applies in particular to the growth areas of sustainability and
+          retirement pensions.</q
         >
         <div class="gap-6 flex items-center">
-          ${img()}
+          <img
+            class="w-16 h-16 rounded-full overflow-hidden object-cover"
+            src="./placeholders/images/HansJoachimReinke.png"
+            alt="Hans Joachim Reinke"
+          />
           <div>
-            <p class="sd-paragraph font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm">Job Title</p>
+            <p class="sd-paragraph font-bold mb-1">Hans Joachim Reinke</p>
+            <p class="sd-paragraph sd-paragraph--size-sm">Chairman of the Board</p>
           </div>
         </div>
-        ${information(false, '3xl')}
       </div>
-
+      <p class="sd-display mb-6">size xl</p>
       <div class="text-left text-black">
-        <q class="block mb-4 text-xl leading-tight">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q>
+        <q class="block mb-4 text-xl leading-tight"
+          >The environment is becoming increasingly challenging. Despite all the crises, the opportunities are greater
+          than the risks in the long term – this applies in particular to the growth areas of sustainability and
+          retirement pensions.</q
+        >
         <div class="gap-4 flex items-center">
-          ${img()}
+          <img
+            class="w-16 h-16 rounded-full overflow-hidden object-cover"
+            src="./placeholders/images/HansJoachimReinke.png"
+            alt="Hans Joachim Reinke"
+          />
           <div>
-            <p class="sd-paragraph font-bold mb-1">Jane Miller</p>
-            <p class="sd-paragraph sd-paragraph--size-sm">Job Title</p>
+            <p class="sd-paragraph font-bold mb-1">Hans Joachim Reinke</p>
+            <p class="sd-paragraph sd-paragraph--size-sm">Chairman of the Board</p>
           </div>
         </div>
-        ${information(false, 'xl')}
-      </div>
-    </div>`
+      </div>`
 };
 
-/**
- * This version, called 'inverted', maintains the flexibility of the quote pattern. It primarily
```

Comment:
why did we remove all the information on the quote? as we don't have a component but only the template, I see a need for some further information on how to use it.